.ci/magic-modules/release-ansible.sh

```diff
@@ -2,5 +2,27 @@
 
 set -x
 
+# Install dependencies for Template Generator
+pushd "magic-modules-gcp"
+bundle install
+
+# Setup SSH keys.
+
+# Since these creds are going to be managed externally, we need to pass
+# them into the container as an environment variable.  We'll use
+# ssh-agent to ensure that these are the credentials used to update.
+set +x
+echo "$CREDS" > ~/github_private_key
+set -x
+chmod 400 ~/github_private_key
+
+ssh-agent bash -c "ssh-add ~/github_private_key; git submodule update --force --init"
+popd
+
 pushd "magic-modules-gcp/build/ansible"
+# Setup Git config.
+git config --global user.email "alexstephen@google.com"
```

Comment:
Should this be a parameter?  I do not love hard-coding human names and email addresses.

Reply:
I'm actually having a change of heart. What do you think about having the magician take credit for these commits?

Reply:
Sounds good to me - it's good karma to give credit to the robots.